scaffold/github/workflows/ComposerLockDiff.yml

```diff
@@ -17,46 +17,33 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          fetch-depth: 2
+          fetch-depth: 0
 
-      - uses: actions/cache@v4
+      - name: Check if composer.lock was changed
+        id: composer-lock-changed
+        uses: tj-actions/changed-files@v44
```

Comment:
@deviantintegral suggests that we pin to a commit hash so future releases don't unexpectedly break our builds.